refactor(react-scheduler): migrate to TS (#1844)
DevExpress/devextreme-reactive: packages/dx-react-scheduler/docs/reference/scheduler.md

------|------|------------
children? | ReactNode | A React node used to render the root layout.

### SchedulerTime

> Describes a scheduler time type.

Reviewer: ```suggestion
The type of date-time values in the Scheduler.
```

Type: `Date | string | number`

### FormatterFn

> Describes a function that format a date by locale.

Reviewer: ```suggestion
A function that formats dates according to the set locale.
```
